src/cascadia/Remoting/Monarch.cpp

```diff
@@ -733,19 +733,27 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
         {
             args.FoundMatch(false);
             uint64_t windowId = 0;
-            // If no name was provided, then just summon the MRU window.
-            if (searchedForName.empty())
+            if (!args.WindowID())

```

Comment:
`WindowID` is a `UInt64`. Do you want to make it an `IReference<UInt64>`? Or is 0 a special value (I think it might be, actually). Either way, put a comment in here if that's the case please.

Reply:
You could probably rewrite this slightly by...
1. removing the else branch
2. just set `windowID` to whatever is given to us from the `args`

Reply:
Oh yes you're right, 0 is a special value so I should be using `IReference` instead. I was intending `WindowID` to be an alternative to specifying what window you want to summon since right now I can only really pass in a window name.